template/src/components/dialog-about/index.js

```diff
@@ -67,6 +68,16 @@ const About = (props) => {
         >
           {`Version v${appVersion}. Powered by WebCatalog.`}
         </Typography>
+        <Typography variant="body2" className={classes.version}>
+          {versions.map(({ name, version }) => (
+            <div>
```

Comment:
for list in React, you should assign a `key` to every item. In this case, it's not that useful but in many other cases, it'll help avoiding many bugs.

Reply:
Oh, I thought eslint will warn me...